lib/hipchat/room.rb

```diff
@@ -230,11 +230,12 @@ def topic(new_topic, options = {})
     #                (default "JSON")
     def history(options = {})
 
-      options = { 
-        :date => 'recent', 
-        :timezone => 'UTC', 
-        :format => 'JSON', 
-        :'max-results' => 100 
+      options = {
+        :date => 'recent',
+        :timezone => 'UTC',
+        :format => 'JSON',
+        :'max-results' => 100,
```

Comment:
The v1 api doesn't respect these fields. With this approach, v2 would get the behavior. Should we throw a InvalidApiVersion on the v1 case instead?